docs/usage/quickstart.rst

```diff
@@ -145,20 +145,21 @@ Similarly, we can list all deployed Helm_ applications::
     nginx-ingress           3               Wed Apr 25 23:09:09 2018        DEPLOYED        nginx-ingress-0.11.1            kube-ingress
     prometheus-operator     3               Wed Apr 25 23:09:14 2018        DEPLOYED        prometheus-operator-0.0.15      kube-ops
 
+
 .. _Helm: https://www.helm.sh
 
-Access to dashboard, Grafana and Kibana
+Access to Dashboard, Grafana and Kibana
 ---------------------------------------
 Once the cluster is running, you can access the `Kubernetes dashboard`_,
-Grafana_ metrics and Kibana_ logs from your browser.
+Grafana_ metrics, and Kibana_ logs from your browser.
 
-To access the Kubernetes dashboard, first create a secure tunnel into your
-cluster by running ``kubectl proxy``. Then, while the tunnel is up and running,
+To access the Kubernetes dashboard, create a secure tunnel into your
+cluster by running ``kubectl proxy``. When the tunnel is up and running,
```

Comment:
I think 'while' is more suited here: we don't want users to close their terminal running `kubectl proxy` and then assume things to still work.

Reply:
done.